src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -189,8 +207,47 @@ class BaseOptionsSelector extends Component {
         return defaultIndex;
     }
 
+    /**
+     * When pagination is enabled,
+     * maps sections to render only allowed count of them.
+     *
+     * @returns {Objects[]}
+     */
+    sliceSections() {
+        if (!_.isNumber(CONST.OPTIONS_SELECTOR_PAGE_LIMITATION)) {
```

Comment:
What is the purpose of this check?

Reply:
Actually, no purpose. It is an echo from the past when it was a prop. Good catch!

Reply:
Done - 49154ff.